packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -170,12 +212,16 @@ describe('Update quantity', function () {
         // update item quantity
         fireEvent.pointerDown(incrementButton)
 
-        await waitFor(() => {
-            expect(within(cartItem).getByDisplayValue('3'))
-        })
+        await waitFor(
+            () => {
+                expect(within(cartItem).getByDisplayValue('3'))
+            },
+            {timeout: 5000}
+        )
     })
 
-    test.skip('Can update item quantity from product view modal', async () => {
+    test('Can update item quantity from product view modal', async () => {
+        jest.setTimeout(30000)
```

Comment:
Let's take a look at this together some time soon, but we can merge this for now / move ahead with the PR